packages/@tinacms/cli/src/next/commands/dev-command/index.ts

```diff
@@ -0,0 +1,263 @@
+import { Command, Option } from 'clipanion'
+import fs from 'fs-extra'
+import path from 'path'
+import chokidar from 'chokidar'
+import { buildSchema, getASTSchema, Database } from '@tinacms/graphql'
+import { ConfigManager } from '../../config-manager'
+import { devHTML } from './html'
+import { logger, summary } from '../../../logger'
+import { createDevServer } from './server'
+import { Codegen } from '../../codegen'
+import chalk from 'chalk'
+import { startSubprocess2 } from '../../../utils/start-subprocess'
+import { createAndInitializeDatabase, createDBServer } from '../../database'
+
+export class DevCommand extends Command {
+  static paths = [['dev'], ['server:start']]
+  port = Option.String('-p,--port', '4001', {
+    description: 'Specify a port to run the server on. (default 4001)',
+  })
+  subCommand = Option.String('-c,--command', {
+    description: 'The sub-command to run',
+  })
+  rootPath = Option.String('--rootPath', {
+    description:
+      'Specify the root directory to run the CLI from (defaults to current working directory)',
+  })
+  watchFolders = Option.String('-w,--watchFolders', {
+    description:
+      'a list of folders (relative to where this is being run) that the cli will watch for changes',
+  })
+  verbose = Option.Boolean('-v,--verbose', false, {
+    description: 'increase verbosity of logged output',
+  })
+  noWatch = Option.Boolean('--noWatch', false, {
+    description: "Don't regenerate config on file changes",
+  })
+  noSDK = Option.Boolean('--noSDK', false, {
+    description: "Don't generate the generated client SDK",
+  })
+  noTelemetry = Option.Boolean('--noTelemetry', false, {
+    description: 'Disable anonymous telemetry that is collected',
+  })
+
+  static usage = Command.Usage({
+    category: `Commands`,
+    description: `Builds Tina and starts the dev server`,
+    examples: [
+      [`A basic example`, `$0 dev`],
+      [`A second example`, `$0 dev --rootPath`],
+    ],
+  })
+
+  async catch(error: any): Promise<void> {
+    logger.error('Error occured during tinacms dev')
+    if (this.verbose) {
```

Comment:
Two things here

1. I think this should log even if verbose is not set. (Since the default is false)

Leaving someone with just "Error occured during tinacms dev" by defualt make it really heard to debug IMO. 

2. Are we stopping the sub process when TinaCMS dev has an error? (I just got into a state where the sub process was running but TinaCMS dev errored)